awx/ui_next/src/components/Lookup/HostFilterLookup.jsx

```diff
@@ -111,13 +111,13 @@ const buildSearchColumns = () => [
 
 function HostFilterLookup({
   helperTextInvalid,
-
   isValid,
   isDisabled,
   onBlur,
   onChange,
   organizationId,
   value,
+  displaySetTypeNot,
```

Comment:
I don't love the naming, as I'm not sure I'd be able to guess (correctly) what it did without looking at the code.

Maybe `enableNegativeFiltering` ?

Reply:
I did not love that name either. Thanks. I will update.